tests/memory-usage.rs

```diff
@@ -54,9 +58,9 @@ fn input_to_tracked_tuple<'db>(
 fn test() {
     let db = salsa::DatabaseImpl::new();
 
-    let input1 = MyInput::new(&db, 1);
-    let input2 = MyInput::new(&db, 2);
-    let input3 = MyInput::new(&db, 3);
+    let input1 = MyInput::new(&db, String::with_capacity(100));
+    let input2 = MyInput::new(&db, String::with_capacity(100));
+    let input3 = MyInput::new(&db, String::with_capacity(100));
```

Comment:
You should make the fields different to ensure they aren't all interned to the same ID.

Reply:
Okay I did that, but I still can't make sense of the numbers.

Reply:
Which part doesn't make sense? It looks fine to me. The reason for the metadata size increase is alignment (previously the `u32` field could fit into the padding).

Reply:
Why is it 522? It should be 400, no?

Reply:
The capacity is actually exact here, it just also includes the stack size of the strings: `250+150+50+(24*3) = 522`.